app/views/goldencobra/admin/articles/_layout_sidebar.html.erb

```diff
@@ -21,7 +21,7 @@
     <h5>
       <%= I18n.t(@article.article_type.split(' ').first.to_s.strip, :scope => [:activerecord, :models, "goldencobra/article"], :default => "Layout der Seite") %>
     </h5>
-    <%= f.select :template_file, Goldencobra::Article.templates_for_select, :include_blank => false %>
+    <%= f.select :template_file, @article.articletype.templates.map { |t| [t.title, t.layout_file_name]}, :include_blank => false %>
```

Comment:
Hier überlege ich gerade. Nach den bisherigen Logiken müsste einem Artikel entweder ein Template zugeordnet werden, oder ein Layout-File, oder?! Aber in jedem Fall kein `template_file`. Das würde einen Change für alle Artikel in der Datenbank bedeuten, richtig? Also Umbenennung des Namen der Spalte. Wollen wir das? Oder wollen wir das vllt. wenn erst in 3.0?

Reply:
Genau! Das Model Template sollte nun (mit der Zeit) mehr Logic und Macht bekommen und die Zuordnung von einem Artikel sollte dann zu einem Template erfolgen und nicht mehr nur den String des verwendeten Layouts speichern. 
Das Refactoring dahingehend steht dann als nächstes an.